### PR TITLE
Created my first Goodie

### PR DESCRIPTION
**Pull request by gjskha**
**What does your Instant Answer do?**

It tells the world that gjskha is awesome.

**What problem does your Instant Answer solve (Why is it better than organic links)?**

It teaches me how t create a DDG Goodie.

**What is the data source for your Instant Answer? (Provide a link if possible)**

Inline

**Why did you choose this data source?**

I wanted to complete the DDG tutorial 

**Are there any other alternative (better) data sources?**

No.

**What are some example queries that trigger this Instant Answer?**

duckduckhack gjskha

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

None.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

No.

**Which existing Instant Answers will this one supercede/overlap with?**

None.

**Are you having any problems? Do you need our help with anything?**

None.

**Where did you hear about DuckDuckHack? (For first time contributors)**

Can't recall.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

## Checklist
Please place an 'X' where appropriate.

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [X] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/gjskha.pm [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+primary_example_queries "duckduckhack gjskha";
+category "special";
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/gjskha.pm";
+attribution github => ["https://github.com/gjskha", "gjskha"];
+          
+triggers start => "duckduckhack gjskha";
+
+handle remainder => sub {
+     return if $_;
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/gjskha.pm]: Can you fix the indentation here?

**Comment by javathunderman** [comment]
@mintsoft Seems like there's another issue with Travis CI...

**Comment by mintsoft** [comment]
@javathunderman No idea what's up with Travis there, it's OK on 5.18 however.

@gjskha Congratulations on your first goodie!

**Comment by moollaza** [comment]
@gjskha congrats! It's live: https://duckduckgo.com/?q=duckduckhack+gjskha&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Pull request merged**